client/lib/plugins/store.js

```diff
@@ -314,7 +314,8 @@ PluginsStore.dispatchToken = Dispatcher.register( function( payload ) {
 	switch ( action.type ) {
 		case 'RECEIVE_PLUGINS':
 			if ( action.error ) {
-				// bail if there is an error since there will be nothing to update
+				updatePlugins( action.site, [] );
```

Comment:
Yeah that makes sense. For some reason I thought we were trying to run the update Plugins action.